packages/react/src/errorboundary.tsx

```diff
@@ -66,6 +66,13 @@ const INITIAL_STATE = {
   eventId: null,
 };
 
+function setCause(error: Error & { cause?: Error }, cause: Error): void {
```

Comment:
Unlikely that it is gonna happen but this could infinitely recurse. Are we worried about that?

Reply:
you are right, let me be a little more defensive and track the visited nodes.